src/App.tsx

```diff
@@ -1,61 +1,109 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
 
 export const App = () => {
+  const [todos, setTodos] = useState<Todo[]>(todosFromServer);
+
+  const [userIdError, setUserIdError] = useState(false);
+  const [inputTitleError, setInputTitleError] = useState(false);
+
+  const [userId, setUserId] = useState(0);
+  const [inputTitle, setInputTitle] = useState('');
+
+  const resetFunc = () => {
+    setUserId(0);
+    setInputTitle('');
+  };
+
+  const handleInputChange = (newInput: string) => {
+    setInputTitleError(false);
```

Comment:
need to set errors state to `false` only after first touch, not after every character